### PR TITLE
feat: bump template to use forma v4

### PR DESCRIPTION
**Pull request by massao**

### CODE REVIEW

**Review by gui-santos — APPROVED** [review]

**Review by bgutsol — APPROVED** [review]

File: packages/contentful--cra-template-create-contentful-app/template.json [hunk 1]
```diff
@@ -2,9 +2,8 @@
   "package": {
     "dependencies": {
       "@contentful/app-sdk": "^4.3.0",
-      "@contentful/forma-36-fcss": "^0.3.2",
-      "@contentful/forma-36-react-components": "^3.90.7",
-      "@contentful/forma-36-tokens": "^0.11.0",
+      "@contentful/f36-components": "latest",
+      "@contentful/f36-tokens": "latest",
```
Comment (andipaetzold) [inline packages/contentful--cra-template-create-contentful-app/template.json:6]: The `package.json` should always have specific version numbers. We don't want to create new apps using `latest`.

**Review by andipaetzold — CHANGES_REQUESTED** [review]
This PR must be merged with a commit message other than `chore:`. Using `chore`, no new version will be released.

File: packages/contentful--cra-template-create-contentful-app/template/src/components/ConfigScreen.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React, { useCallback, useState, useEffect } from 'react';
 import { AppExtensionSDK } from '@contentful/app-sdk';
 import { PlainClientAPI } from 'contentful-management';
-import { Heading, Form, Workbench, Paragraph } from '@contentful/forma-36-react-components';
-import styles from './ConfigScreen.module.css';
+import { Heading, Form, Paragraph, Flex } from '@contentful/f36-components';
+import { css } from 'emotion';
```
Comment (andipaetzold) [inline packages/contentful--cra-template-create-contentful-app/template/src/components/ConfigScreen.tsx:5]: `emotion` is not part of the template anymore. Please keep using the CSS module

**Review by andipaetzold — CHANGES_REQUESTED** [review]

File: packages/contentful--cra-template-create-contentful-app/template/src/components/ConfigScreen.tsx (hunk 2 above)
Comment (suevalov) [reply packages/contentful--cra-template-create-contentful-app/template/src/components/ConfigScreen.tsx:5]: @andipaetzold `emotion` is a recommended way to style Forma v4 components and I think we need to re-evaluate the decision

Comment (andipaetzold) [reply packages/contentful--cra-template-create-contentful-app/template/src/components/ConfigScreen.tsx:5]: `emotion` was only used in this file and we saw that as an opportunity to get rid of a dependency and make the bundle smaller. If we want to reintroduce `emotion` with this PR, we'd need to add it back to `template.json`.

Comment (massao) [reply packages/contentful--cra-template-create-contentful-app/template/src/components/ConfigScreen.tsx:5]: I believe that we want to add it back as it's the recommended library to be used with forma v4 components, right @burakukula and @suevalov ??

Comment (suevalov) [reply packages/contentful--cra-template-create-contentful-app/template/src/components/ConfigScreen.tsx:5]: @massao yes, I'd do that

**Review by andipaetzold — CHANGES_REQUESTED** [review]
Please remove the `yarn.lock` again

**Review by suevalov — APPROVED** [review]

**Review by andipaetzold — APPROVED** [review]

**Pull request merged**